hardware/raspberrypi/bcm2837b0/README.md

```diff
@@ -0,0 +1,16 @@
+# BCM2837B0
+
+This is the Broadcom chip used in the Raspberry Pi 3B+. The underlying architecture of the BCM2837B0 is identical to the BCM2836 and BCM2837. The only significant difference is the replacement of the ARMv7 quad core cluster with a quad-core ARM Cortex A53 (ARMv8) cluster.
+
```

Comment:
This isn't quite right. The B0 is basically exactly the same as the A0, it's just the packaging that is different, which is one reason why the thermals are better. There is no ARM core change between the A0 and the B0.